### PR TITLE
AUT-2930 - Create initial template to deploy frontend

### PR DESCRIPTION
**Pull request by monhaque**
## What

Create initial template to deploy frontend

## Checklist

- [ ] Performance analyst has been notified of the change.

- [ ] A UCD review has been performed.

- [ ] Any necessary changes to the [acceptance tests](https://github.com/govuk-one-login/authentication-acceptance-tests) have been made.

- [ ] Documentation has been updated to reflect these changes.

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -89,14 +167,231 @@ Globals:
...
+          - !GetAtt FrontendECSService.Name
+      ScalableDimension: ecs:service:DesiredCount
+      ServiceNamespace: ecs
+      StepScalingPolicyConfiguration:
+        AdjustmentType: PercentChangeInCapacity
+        Cooldown: 240
+        MetricAggregationType: Average
+        MinAdjustmentMagnitude: 5
+        StepAdjustments:
+          - ScalingAdjustment: -50
```
Comment (pskushwaha1) [inline template.yaml:339]: There is new Adjustment recently pushed for Scale In policy  & Scale In Alarm if you want to use that

File: template.yaml [hunk 2]
```diff
@@ -89,14 +167,231 @@ Globals:
...
+      AlarmName: !Sub "${FrontendECSCluster}/${FrontendECSService.Name}-AlarmScaleDown"
+      AlarmDescription: >
+        Metric alarm to trigger ECS scale down
+      AlarmActions:
+        - !Ref FrontendAutoScalingPolicyScaleIn
+      ComparisonOperator: LessThanThreshold
+      EvaluationPeriods: 5
+      MetricName: CPUUtilization
+      Namespace: AWS/ECS
+      Period: 60
```
Comment (pskushwaha1) [inline template.yaml:268]: here is new Adjustment recently pushed for Scale In policy & Scale In Alarm if you want to use that

**Review by pskushwaha1 — APPROVED** [review]

File: template.yaml (hunk 1 above)
Comment (monhaque) [reply template.yaml:339]: Applied

File: template.yaml (hunk 2 above)
Comment (monhaque) [reply template.yaml:268]: Applied

**Review by pskushwaha1 — APPROVED** [review]

**Pull request merged**